slices/xz-utils.yaml

```diff
@@ -0,0 +1,36 @@
+package: xz-utils
+
+essential:
+  - xz-utils_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - liblzma5_libs
+    contents:
+      /usr/bin/lzmainfo:
+      /usr/bin/unxz:
+      /usr/bin/xz:
+      /usr/bin/xzcat:
+  utils:
+    essential:
+      - base-files_bin
+      - base-files_tmp
```

Comment:
why is this depending on /tmp?

Reply:
Looking around with grep, the xz diff utils make use of the /tmp directory.